apps/src/studio/studio.js

```diff
@@ -3934,10 +3970,35 @@ Studio.setSprite = function (opts) {
     spriteWalk.setAttribute('height', sprite.drawHeight * sprite.frameCounts.walk); // 1200
   }
 
+  // Set up movement audio for the selected sprite (should be preloaded)
+  Studio.movementAudioOptions = Studio.movementAudioOptions || {};
```

Comment:
That's a good point, I'll take care of this.

* [ ] TODO: Initialize Studio.movementAudioOptions in Studio.init()